### ISSUE TITLE
Target `netX.0-macos` for `IOBluetooth` package

### ISSUE THREAD

**Issue by ShortDevelopment** [inthehand/32feet#322]
It's not possible to use the (latest) `InTheHand.IOBluetooth` nuget package from a dotnet app targeting `net7.0-macos`.
You maybe could just modify the existing project file for mac-catalyst to use `net6.0-macos`.
https://github.com/inthehand/32feet/blob/3c0436f67778aa1f3a022e9c122749a275971d9b/InTheHand.IOBluetooth/InTheHand.IOBluetooth.macCatalyst.csproj#L1-L17

**Comment by peterfoot** [comment]
Yes, currently the nuget is just the Xamarin Mac binding. I will get around to completing the .NET 6/7 project soon but it's been on hold due to a lack of time and competing priorities
